From a1c4718d9db069ab088914ec983af8125eba3ac6 Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Fri, 31 Mar 2017 09:34:27 -0400
Subject: [PATCH] Issue 49177 - Fix pkg-config file

Description:  Need to remove the slash in front of the package name

https://pagure.io/389-ds-base/issue/49177

Reviewed by: lslebodn & wibrown (Thanks!!)
---
 Makefile.am  | 4 ++--
 configure.ac | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 982dd28..485a460 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -261,7 +261,7 @@ sampledatadir = $(datadir)@sampledatadir@
 systemschemadir = $(datadir)@systemschemadir@
 propertydir = $(datadir)@propertydir@
 schemadir = $(sysconfdir)@schemadir@
-serverdir = $(libdir)@serverdir@
+serverdir = $(libdir)/@serverdir@
 serverplugindir = $(libdir)@serverplugindir@
 taskdir = $(datadir)@scripttemplatedir@
 systemdsystemunitdir = @with_systemdsystemunitdir@
@@ -276,7 +276,7 @@ infdir = $(datadir)@infdir@
 mibdir = $(datadir)@mibdir@
 updatedir = $(datadir)@updatedir@
 pkgconfigdir = $(libdir)/pkgconfig
-serverincdir = $(includedir)@serverincdir@
+serverincdir = $(includedir)/@serverincdir@
 
 defaultuser=@defaultuser@
 defaultgroup=@defaultgroup@
diff --git a/configure.ac b/configure.ac
index 8172bab..51c4414 100644
--- a/configure.ac
+++ b/configure.ac
@@ -361,9 +361,9 @@ else
   # relative to datadir
   updatedir=/$PACKAGE_NAME/updates
   # relative to libdir
-  serverdir=/$PACKAGE_NAME
+  serverdir=$PACKAGE_NAME
   # relative to includedir
-  serverincdir=/$PACKAGE_NAME
+  serverincdir=$PACKAGE_NAME
   # relative to libdir
   serverplugindir=/$PACKAGE_NAME/plugins
   # relative to datadir
-- 
2.9.3

